docker-compose.yml

```diff
@@ -14,6 +14,8 @@ services:
     volumes:
       - './:/srv/pim'
       - '${HOST_COMPOSER_HOME:-~/.composer}:/var/www/.composer'
+      - './config/services/akeneo-custom-php.ini:/etc/php/7.3/cli/conf.d/9a-akeneo-custom.ini'
```

Comment:
Are we really using 7.3?

Reply:
Affirmative:
![image](https://user-images.githubusercontent.com/26855/99470847-4f29b300-28fa-11eb-8232-a109d48b4cc4.png)
 - https://docs.akeneo.com/4.0/install_pim/manual/system_requirements/system_requirements.html